app/Http/Controllers/RoleController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Role;
+use Illuminate\Http\Request;
+
+class RoleController extends Controller
+{
+    public function index(Request $request)
+    {
+        $this->authorize('lists', Role::class);
```

Comment:
We now use viewAny (laravel default) so for consistency let's use viewAny here as well.
```suggestion
        $this->authorize('viewAny', Role::class);
```